detectors/node/opentelemetry-resource-detector-azure/src/detectors/AzureFunctionsDetector.ts

```diff
@@ -49,28 +50,23 @@ const AZURE_FUNCTIONS_ATTRIBUTE_ENV_VARS = {
 class AzureFunctionsDetector implements DetectorSync {
   detect(): IResource {
     let attributes = {};
-    const functionName = process.env[WEBSITE_SITE_NAME];
+    const serviceName = process.env[WEBSITE_SITE_NAME];
     const functionVersion = process.env[FUNCTIONS_VERSION];
```

Comment:
is it used for anything now that the attribute is removed?

Reply:
Yes, it's now correctly reported under the semantic resource attribute `SERVICE_NAME`, instead of the `FAAS_NAME` since this represents the name of the app service instance that azure functions runs within, not the name of the azure function that contains this worker.

Reply:
I was actually referring to the `cons functionVersion` variable, which used to be populated into `SEMRESATTRS_FAAS_VERSION`, but now we don't use it for anything (except the `if` condition below that does not access it anymore).
If I am not missing anything, can this code line be safely removed?

Reply:
Thanks! I never worked with azure function, so it doesn't mean a lot to me, but I suggest adding a comment next to this line to explain why it's there and why it's important, for the future maintainers and readers.

Reply:
Makes sense. I've added a note to make this more clear.